testdata/project-v3/go.mod

```diff
@@ -10,5 +10,5 @@ require (
 	k8s.io/apimachinery v0.19.2
 	k8s.io/client-go v0.19.2
 	sigs.k8s.io/controller-runtime v0.7.2
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20210113160450-b84d99da0217
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200522144838-848d48e5b073
```

Comment:
It means that a release was made. However, in the past, we fixed that by pinning a declarative commit-version. So, it might be a regression.

Reply:
No, we are still pinning but there is a check for go/v2 or go/v3 because different SHAs are pinned for each case and the check seems to be broken.